pkg/logging/options.go

```diff
@@ -175,7 +175,7 @@ func ParseHTTPOptions(flagDecision string, reqLogConfig *extflag.PathOrContent)
 
 	// Check if the user enables request logging through flags and YAML.
 	if len(configYAML) != 0 && len(flagDecision) != 0 {
-		return logOpts, fmt.Errorf("Both log.request.decision and request.logging has been enabled. Please use one of the flags!")
+		return logOpts, fmt.Errorf("both log.request.decision and request.logging has been enabled. Please use one of the flags")
```

Comment:
```suggestion
		return logOpts, fmt.Errorf("both log.request.decision and request.logging has been enabled, please use only one of the flags")
```

As @squat said this is a bit tricky, but I'll still prefer no-caps to be consistent. After all this is going to be printed in a structured format `err="both log......"` so it's better to be consistent.